build_tools/cirrus/arm_wheel.yml

```diff
@@ -33,7 +33,7 @@ linux_arm64_wheel_task:
         CIBW_BUILD: cp312-manylinux_aarch64
     - env:
         CIBW_BUILD: cp313-manylinux_aarch64
-        # TODO remove next line when Python 3.13 is relased and add
+        # TODO remove next line when Python 3.13 is released and add
```

Comment:
BTW: Python 3.13 HAS been released.

Reply:
So you are saying we have TODO in our code and we haven't found time to act on them, no kidding :upside_down_face:

I think this particular TODO note is going to not matter because of https://github.com/scikit-learn/scikit-learn/pull/30658

Reply:
Actually, we should wait for https://github.com/scikit-learn/scikit-learn/pull/30660#issuecomment-2598586573.

Reply:
I meant to say that I could change the contents of the TODO in this PR.